Maestro/subscriptions.json

```diff
@@ -1175,6 +1181,29 @@
         }
       }
     },
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/build.semaphore"
+      ],
+      "action": "source-build-general",
+      "actionArguments": {
+        "vsoSourceBranch": "dev/release/2.1",
+        "vsoBuildParameters": {
+          "ScriptFileName": "build.cmd",
+          "Arguments": [
+            "/t:UpdateToRemoteDependencies",
+            "/t:SubmitPullRequestIfChanged",
+            "/p:GitHubUser=dotnet-maestro-bot",
+            "/p:GitHubEmail=dotnet-maestro-bot@microsoft.com",
+            "/p:GitHubAuthToken=`$(`$Secrets['DotNetMaestroBotGitHubToken'])",
+            "/p:ProjectRepoOwner=dotnet",
+            "/p:ProjectRepoName=source-build",
+            "/p:ProjectRepoBranch=dev/release/2.1",
```

Comment:
That's an interesting naming scheme, what does it signify?

Reply:
At the moment the `dev/` prefix is a historical thing from the branching pattern for earlier releases. I believe we used it in the past to iterate on infrastructure while others did work with the existing infra to get a release out the door. Normally that would be `master`, but both branches used `release/...` submodules so that could have been misleading.